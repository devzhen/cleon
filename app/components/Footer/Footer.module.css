.footer {
  grid-column: 2 span;
  height: 44px;
  width: 100%;
  display: flex;
  align-items: center;
}

.footerBg {
  height: 41px;
  margin-top: auto;
  padding-left: 22px;
  display: flex;
  flex: 1;
  align-items: center;
  background-color: var(--color-main-background);
}

.footer a {
  display: flex;
  align-items: center;
  text-decoration: none;
  cursor: pointer;
}

.footer span {
  margin-left: 2px;
  font-size: 14px;
  color: var(--color-text-area-color);
}
