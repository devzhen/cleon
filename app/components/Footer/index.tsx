import Image from 'next/image';

import styles from './Footer.module.css';

export default function Footer() {
  console.log(111, process.env.NEXT_PUBLIC_GITHUB_REPO);
  console.log(222, process.env.GITHUB_REPO);

  return (
    <footer className={styles.footer}>
      <div className={styles.footerBg}>
        <a href={process.env.NEXT_PUBLIC_GITHUB_REPO}>
          <Image src="/github-mark.png" width={24} height={24} alt="Img" draggable={false} />
          <Image src="/github-logo.png" width={54} height={24} alt="Img" draggable={false} />
          <span>go to the repo</span>
        </a>
      </div>
    </footer>
  );
}
