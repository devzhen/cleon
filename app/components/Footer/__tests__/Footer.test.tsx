import { render } from '@testing-library/react';

import Footer from '..';

describe('Footer tests', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
