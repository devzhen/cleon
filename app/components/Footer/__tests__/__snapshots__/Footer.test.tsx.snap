// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests matches the snapshot 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="footerBg"
    >
      <a>
        <img
          alt="Img"
          data-nimg="1"
          decoding="async"
          draggable="false"
          height="24"
          loading="lazy"
          src="/_next/image?url=%2Fgithub-mark.png&w=48&q=75"
          srcset="/_next/image?url=%2Fgithub-mark.png&w=32&q=75 1x, /_next/image?url=%2Fgithub-mark.png&w=48&q=75 2x"
          style="color: transparent;"
          width="24"
        />
        <img
          alt="Img"
          data-nimg="1"
          decoding="async"
          draggable="false"
          height="24"
          loading="lazy"
          src="/_next/image?url=%2Fgithub-logo.png&w=128&q=75"
          srcset="/_next/image?url=%2Fgithub-logo.png&w=64&q=75 1x, /_next/image?url=%2Fgithub-logo.png&w=128&q=75 2x"
          style="color: transparent;"
          width="54"
        />
        <span>
          go to the repo
        </span>
      </a>
    </div>
  </footer>
</div>
`;
